src/main/java/com/zendesk/maxwell/RowMap.java

```diff
@@ -105,7 +105,7 @@ private String pkToJsonHash() throws IOException {
 				if ( data.containsKey(pk) )
 					pkValue = data.get(pk);
 
-				g.writeObjectField("pk." + pk, pkValue);
+				g.writeObjectField("pk." + pk.toLowerCase(), pkValue);
```

Comment:
if it's lower casing `pk` here, why is `Table` `normalizedPKColumnNames` needed? (couldn't find other places where `Table#getPKList` is being used)